back/src/app/controllers/user.ts

```diff
@@ -1,14 +1,30 @@
-
 import { User } from '../../app/models/user'
 import {Request, Response} from 'express'
-import { validationResult } from 'express-validator'
+import {validate} from 'class-validator'
 
 export default class userController {
 
 	static create(req: Request, res: Response) {
 		const user = new User();
+		user.email =req.body.email;
+		user.pseudo = req.body.pseudo;
+		user.firstname = req.body.firstname;
+		user.lastname = req.body.lastname;
+		user.password = req.body.password;
+		user.imageUrl = req.body.img;
+		validate(user).then(errors => { // errors is an array of validation errors
```

Comment:
slice user assignation and validation user in two function. we might need validation later in other context

Reply:
ok.